data_engineering/common/views/base.py

```
@@ -6,6 +6,12 @@
 
 
 class PaginatedListView(View):
+    def get_fields(self):
+        return self.get_field_types_from_column_types(self.pipeline._l1_data_column_types)
```

I don't think there should be any mention of pipelines or l1 here, it should just raise a notimplementederror error